Source/GoBot/ViewDatabase.swift

```diff
@@ -233,6 +238,7 @@ class ViewDatabase {
             }
         }
 
+        self.currentUser = user
```

Comment:
Maybe it doesn't change anything, but for completeness we should set it to null in func close()

Reply:
Yeah good call.